tests/unittest/auto_deploy/multigpu/smoke/test_ad_allreduce_strategies.py

```
@@ -0,0 +1,310 @@
+            # Create dummy tensors to trigger kernel JIT compilation
+            with torch.no_grad():
+                device = torch.device("cuda:0")
+
+                # Trigger page kernel compilation
+                try:
+                    # Force module loading (this triggers JIT compilation)
+                    _ = flashinfer.page.gen_page_module()
+                except Exception as exc:  # noqa: BLE001
+                    import warnings
+
+                    warnings.warn(f"FlashInfer page-kernel prewarm failed: {exc!r}", RuntimeWarning)
```

Ruff (0.15.21)

[warning] No explicit `stacklevel` keyword argument found

Set `stacklevel=2`

(B028)